index.html

```diff
@@ -5,8 +5,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
 </head>
 <body>
-    
+    <section id="container">
+      <header>
+        <h1 id='page-title'>Weather Report - 🌟 Sharjah 🌟</h1>
+        <h5 id="sub-header">Should you leave your house today? Or nah? 🤔</h5>
+      </header>
+      <section>
+        <article id="temp">
```

Comment:
Throughout your file, you are using the section tag and article tag. You could just use the section tag to separate your information. The article tag would normally be used for something like a blog post. More information is here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article